src/conf.cls.php

```diff
@@ -633,19 +633,33 @@ public function network_update( $id, $val ) {
 	 * @since 1.6
 	 * @access public
 	 * @param  string $role The user role
-	 * @return int       The set value if already set
+	 * @return boolean
 	 */
 	public function in_optm_exc_roles( $role = null ) {
-		// Get user role
-		if ( $role === null ) {
-			$role = Router::get_role();
```

Comment:
If dropped $role assignment in `in_optm_exc_roles`, will this break the existing code?